### ISSUE TITLE
API Connector: Minor UI tidbits

### ISSUE THREAD

**Issue by rhuss** [syndesisio/syndesis#1709]
* The wording "1 dealing with ..." seems redundant and a bit too wordy to me. It does not really provide information and move the relevant info (what its actually 'deals with') to the right:

![image](https://user-images.githubusercontent.com/99080/36748616-a577869a-1bf8-11e8-9218-f15424688bb5.png)

I'd recommend to use something like "1 _CREATE_ operation" (capitalizing to emphasize that this is a kind of type). Not sure what 'tasks' and 'destruction' really means. Also, no online doces explaining the various types are given.

* Everywhere in this section it is talked about an "API Connector" (see e.g. the "Create API Connector" button). Therefor I recommend to use "Create API Connector" also on this screen, for consistencies sake:

![image](https://user-images.githubusercontent.com/99080/36748623-b026554e-1bf8-11e8-8b0e-25b4907793e4.png)

**Comment by rhuss** [comment]
btw, why is there extra spacing between password and username ?

**Comment by amysueg** [comment]
@zregvart @sjcox-rh 
UX team looking at this now and we have some thoughts:

1. For the "review" screen, could we replace the label "dealing with" with "tagged?" So the list would read:

16 operations
   4 tagged updating
   5 tagged editing

....something like that. We think that the user of the API Connector will recognize the standard usage of "tagged" as it is part of Swagger nomenclature.

2. We agree to change the button from "Create Connector" to "Create API Connector"

3. For the correct ordering of fields in the Configuration screen, the proper field order should be supplied via back-end code, made possible by this PR https://github.com/syndesisio/syndesis/pull/1698 

4. Spacing between fields: as this is a model-driven form... this needs more investigation.  @rhuss -- Please create a new issue for this one. We're not sure how to reproduce it, thanks.
